OasisDEX/oasis-borrow
# Rz/open multiply erc20 collaterals

# [Prepare for other collateral types](https://app.clubhouse.io/oazo-apps/story/2389/prepare-for-other-collateral-types-open-multiply-vault-add-allowance-steps)
  
## Changes 👷‍♀️
- add `ALLOWED_MULTIPLY_TOKENS` inside `tokensMetadata`, exclude `lp-token` and `stablecoin`
- fix copy
- bring back storybooks for allowance stages
  
## How to test 🧪
- try to open any non `lp-token` vault, you should now see option to choose `borrow/multiply` for those
- see storybooks stages for allowances
    
## Definition of done ✔️
